src/Components/ContactsList.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import "./ContactsList.css";
+import ContactForm from "./ContactForm.js";
+
+const ContactsList = () => {
+  const [contacts, setContacts] = useState([
+    {
+      firstName: "Ivan",
+      lastName: "Yakushenko",
+      phoneNumber: "+380934869702",
+    },
+    {
+      firstName: "Oleg",
+      lastName: "Bobrov",
+      phoneNumber: "+380973358892",
+    },
+    {
+      firstName: "Irina",
+      lastName: "Rogovskaya",
+      phoneNumber: "+380956699944",
+    },
+  ]);
+
+  const [show, setShow] = useState("true")
+
+  const addContact = (firstName, lastName, phoneNumber) => setContacts([...contacts, { firstName, lastName, phoneNumber }]);
```

Comment:
See the previous comment about `onSubmit`

Now you can change to one argument instead of three

```
const handleSubmit = (payload) => setContacts(prevState => [...prevState, payload]);
```